client/src/hooks/useGetMovies.js

```diff
@@ -1,6 +1,6 @@
 import {useEffect,useState} from 'react'
 import { toast } from 'react-toastify';
-import { fetchMovies } from '../api/netflixApi';
+import { fetchMovies } from '../api/netflixApi.js';
```

Comment:
Revert this change, imports do not need to specify file type. Something is off here